climakitae/data/variable_descriptions.csv

```diff
@@ -0,0 +1,22 @@
+name,description,extended_description
+T2,2m Air Temperature,"Temperature of the air 2 meters above Earth's surface. This is the measure of air temperature used for most modeling applications."
+TOT_PRECIP,Precipitation (total),"Total precipitation. Computed by summing total gridscale precipitation and total cumulus precipitation."

```

Comment:
I think "Precipitation" without the total would suffice for description.

Reply:
@thenaomig @vicford Do either of you have opinions on this? I'm inclined to leave "total" in this description for clarity's sake (since we have the other precip variables as options) but don't feel strongly.